.github/workflows/main.yml

```diff
@@ -0,0 +1,13 @@
+on: [push]
+
+jobs:
+  hello_world_job:
```

Comment:
IMO, you should probably change the identifier of this job.

Reply:
I definitely should